assets/javascipt/game.js

```diff
@@ -0,0 +1,159 @@
+const charArr = [
```

Comment:
first of all, what do you have against ginger. second, great job! I look forward to all your games :)